content/guides/java/tracers.md

```diff
@@ -2,7 +2,67 @@
 title: "Java: Tracers"
 ---
 
-* Setting up your tracer
-* Using the Global Tracer
-* Accessing the Active Span
-* Accessing tracer specific features.
\ No newline at end of file
+#### Setting up your Tracer
+
+Different **Tracer** implementations vary in how and what parameters they receive at initialization time, such as:
+
+* Component name for this application's traces.
+* Tracing endpoint.
+* Tracing credentials.
+* Sampling strategy.
+
+
+For example, initializing the **Tracer** implementation of `Jaeger` looks like this:
+
+```java
+  import io.opentracing.Tracer;
+
+  SamplerConfiguration samplerConfig = new SamplerConfiguration("const", 1);
+  ReporterConfiguration reporterConfig = new ReporterConfiguration(true, null, null, null, null);
+  Configuration config = new Configuration(service, samplerConfig, reporterConfig);
+
+  // Get the actual OpenTracing-compatible Tracer.
+  Tracer tracer = config.getTracer();
+```
+
+One a **Tracer** instance is obtained, it can be used to manually create **Span**, or pass it to existing instrumentation for frameworks and libraries:
+
+```java
+  // OpenTracing Redis client. It can be *any* OT-compatible tracer.
+  Tracer tracer = ...;
+  new TracingJedis(tracer);
+```
+
+#### Global Tracer
+
+In order to not force the user to keep around a **Tracer**, the **io.opentracing.util** artifact includes a helper **GlobalTracer** class implementing the **io.opentracing.Tracer**, which, as the name implies, acts as as a global instance that can be used from anywhere. It works by forwarding all operations to another underlying **Tracer**, that will get registered at some future point.
```

Comment:
Might be better to state the actual artifact coordinates using the standard `g:a[:v]` notation: `io.opentracing:opentracing-util`, possibly with a link to the git repository?

Reply:
I covered this in the `Intro` section. Think we should put them here as well?